src/fluid/prim2con.hpp

```diff
@@ -24,7 +24,7 @@ namespace prim2con {
 KOKKOS_INLINE_FUNCTION
 void signal_speeds(const Real &rho, const Real &u, const Real &p, const Real &bsq,
                    const Real v[], const Real &vsq, const Real &gam1, const Real &alpha,
-                   const Real beta[], const Real gcon[3][3], Real *sig) {
+                   const Real beta[], const Real gcon[3][3], Real *sig) { 
```

Comment:
Looks like you added a trailing space here, which we don't need. No big deal either way though